packages/firestore/test/unit/specs/spec_test_runner.ts

```diff
@@ -120,15 +128,22 @@ class MockConnection implements Connection {
   /** A Deferred that is resolved once watch opens. */
   watchOpen = new Deferred<void>();
 
+  reset(): void {
+    this.watchStreamRequestCount = 0;
+    this.writeStreamRequestCount = 0;
+    this.earlyWrites = [];
+    this.activeTargets = [];
+  }
+
   invoke(rpcName: string, request: any): Promise<any> {
     throw new Error('Not implemented!');
   }
 
-  waitForWriteSend(): Promise<api.Write> {
+  waitForMutationsSend(): Promise<MutationBatch> {
```

Comment:
This should be `waitForWritesSend`

Reply:
With the rename above, this is now waitForWriteRequest.